invenio_vocabularies/contrib/common/ror/datastreams.py

```diff
@@ -36,25 +42,32 @@ def read(self, item=None, *args, **kwargs):
 
         # Call the signposting `linkset+json` endpoint for the Concept DOI (i.e. latest version) of the ROR data dump.
         # See: https://github.com/inveniosoftware/rfcs/blob/master/rfcs/rdm-0071-signposting.md#provide-an-applicationlinksetjson-endpoint
-        headers = {"Accept": "application/linkset+json"}
+        headers = {"Accept": "application/ld+json"}
```

Comment:
minor: I meant that we make both calls for the `linkset+json` (to get the list files and see if there's an `ld+json` format) and then the `ld+json` to get the metadata for the creation date. I know it's two requests (and that the `ld+json` already includes the link to the file), but from a semantics point of view, one cannot expect that the `ld+json` format is available without looking into what comes back in the `linkset+json` response.